LukasChiama/Banka-App
# #165478469 Migrate delete user endpoint from file system to the database

#### What does this PR do?
Migrate delete endpoint to database

#### Description of Task to be completed?

- ensure DELETE request to the endpoint `/api/v1/users/:user_id` is working

- ensure relevant tests for this endpoint are passing

#### How should this be manually tested?
Clone the repo and run `npm run test` from the root. A DELETE request can also be made from Postman. Please note that the route is secure and requires a staff token to be accessed.

#### Any background context you want to provide?
This PR is necessary because of the migration of the code base from using the Node.js file system functions for interacting with data to using a PostgreSQL database. This PR handles the migration of the DELETE user enpoint.

#### What are the relevant pivotal tracker stories?
[#165478469](https://www.pivotaltracker.com/story/show/165478469)

#### Screenshots (if appropriate)
![Screenshot (70)](https://user-images.githubusercontent.com/42539315/56449121-98c0af00-630d-11e9-9616-a3810b2a570b.png)
